sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/implementation/SyncToken.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration.implementation;
+
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.logging.ClientLogger;
+
+/**
+ * SyncToken class requires id, value and sequence number
+ */
+public final class SyncToken {
+    private final ClientLogger logger = new ClientLogger(SyncToken.class);
+
+    private String id, value;
+    private long sequenceNumber;
+    private String syncTokenString;
```

Comment:
Should make these final as they shouldn't change during the lifetime of a `SyncToken`.

Reply:
These still can be made `final`